test/integrity_checks.rb

```diff
@@ -333,31 +333,7 @@ def check_parameter_file_format(tsvpath, n_deps, name)
 
     # Check endline character
     if line.include? "\r\n"
-      # Do not perform other checks if the line is the header
-      if i > 1
-        # Check float format
-        # Remove endline character and split the string into array
-        line = line.split("\r\n")[0].split("\t")
-        # For each non dependency entry check format
-        for j in n_deps..line.length() - 1 do
-          # Check for scientific format
-          if (line[j].include?('e-') || line[j].include?('e+') ||
-              line[j].include?('E-') || line[j].include?('E+'))
-            raise "ERROR: Scientific notation found in '#{name}', line '#{i}'."
-          end
-
-          begin # Try to get the float precision
-            float_precision = line[j].split('.')[1].length()
-          rescue NoMethodError
-            # Catch non floats
-            raise "ERROR: Incorrect non float found in '#{name}', line '#{i}'."
-          end
-          # If float precision is not 6 digits, raise error
-          if float_precision != 6
-            raise "ERROR: Incorrect float precision found in '#{name}', line '#{i}'."
-          end
-        end
-      end
+      next
     else
```

Comment:
There is a small bug here, where by using `next`, the `i += 1` below will not get executed.

Suggest changing this from:
```
    if line.include? "\r\n"
      next
    else
```
to
```
    if not line.include? "\r\n"
```

Reply:
Thanks @shorowit!